dask_kubernetes/operator/controller/controller.py

```diff
@@ -260,6 +254,7 @@ async def daskcluster_create_components(spec, name, namespace, logger, patch, **
 
         worker_spec = spec.get("worker", {})
         data = build_default_worker_group_spec(name, worker_spec, annotations)
+        kopf.adopt(data)
```

Comment:
I removed this at some point because worker groups get adopted to their cluster after creation so that they are cascade deleted regardless of how they were created.